assets/components/svgs/arrowLeftStraight.jsx

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import React from 'react';
+
+// The right-pointing arrow commonly used on CTAs.
+export default function SvgArrowLeftStraight() {
+  const reverseStyle = {
+    transform: 'scale(-1, -1)',
+  };
```

Comment:
Had to put this as an object otherwise flow complained, this is a common get-around (see [here](https://github.com/alex3165/react-mapbox-gl/issues/498))

Reply:
Is it the same as `rotate(180deg)`?

Reply:
Yup. Happy to change it as it is clearer!

Reply:
I think so yeah, though it is a shame you have to duplicate the SVG for that 😐

Reply:
I'm now using CSS to rotate it rather than duplicate it